### PR TITLE
[fix] telegram: Add's comments to message

### PR DESCRIPTION
**Pull request by soloam**
This is a change to handle comments in telegram messages!

Some messages in telegram have comments added to the message! This normally is the description in images or links. This changes appends the comment to the message if available.

This should fix the issue in #1649

### CODE REVIEW

**Comment by 42wim** [comment]
hi @soloam did you test this? the code doesn't build

**Comment by soloam** [comment]
I'm sorry for the confusion, I pushed the wrong branch! I corrected the branch and updated the new change! Discord was ignoring the image caption if a text was in the image! The caption contains important information from the extra! It should be forwarded!

**Comment by 42wim** [comment]
But this change will remove any text ?

File: bridge/discord/webhook.go [hunk 1]
```diff
@@ -82,10 +82,8 @@ func (b *Bdiscord) webhookSend(msg *config.Message, channelID string) (*discordg
 				ContentType: "",
 				Reader:      bytes.NewReader(*fi.Data),
 			}
-			content := ""
-			if msg.Text == "" {
-				content = fi.Comment
-			}
+			content := fi.Comment
```
Comment (42wim) [inline bridge/discord/webhook.go:85]:
```suggestion
			content := fi.Comment
			if msg.Text != "" {
				content = msg.Text
			}
```

Comment (42wim) [reply bridge/discord/webhook.go:85]: can you test with this change, so we only add the comment when there is empty text

**Review by 42wim — CHANGES_REQUESTED** [review]

**Comment by soloam** [comment]
That was the original behavior! The problem is that when it's forwarded (for example telegram writers "forwarded from") or anything is written on the text, the comments are removed.

**Comment by 42wim** [comment]
Ok, as I'm not using telegram I take you for your word :)

**Review by 42wim — APPROVED** [review]

**Pull request merged**